docs/globaloptions.rst

```diff
@@ -8,7 +8,12 @@ and allow you to customize various aspects of the ``rack`` tool at runtime.
 These options may override configuration file or environment variables you have
 set previously, change output format or other aspects of the tool.
 
-For example:
+Display any available command-line flags by appending `--help` to list the
+flags available::
+
+    rack <service> <command> <subcommand> --help
```

Comment:
I think we decided on `rack <service> <subservice> <action> --help` for describing the command format.